python/cudf/cudf/tests/test_parquet.py

```diff
@@ -437,3 +467,20 @@ def test_multifile_warning(datadir):
         expect = pd.read_parquet(fname)
         expect = expect.apply(pd.to_numeric)
         assert_eq(expect, got)
+
+
+# Validates the integrity of the GPU accelerated parquet writer.
+def test_parquet_writer_gpu(tmpdir, simple_gdf):
+    gdf_fname = tmpdir.join("gdf.parquet")
+
+    # Write out the gdf using the GPU accelerated writer
+    simple_gdf.to_parquet(gdf_fname.strpath, engine="cudf")
+    assert os.path.exists(gdf_fname)
+    expect = simple_gdf
+
+    # Read the gdf back in using the reader
+    got = cudf.read_parquet(
+        gdf_fname, columns=["col_int8", "col_int16", "col_int32"]
```

Comment:
we shouldn't have to specify columns here, this could hide us implicitly writing additional columns

Reply:
Makes sense. Removed those.